CHANGES.md

```diff
@@ -2,6 +2,7 @@
 
 ## __NEXT__
 
+* fix(translate): constrain Biopython version to <=1.80 so that BCBio.GFF imports succesfully. [#1152][] (@corneliusroemer)
```

Comment:
This is a user, not developer, focused changelog, so I'd suggest some rewording:

```suggestion
### Bug fixes

* Constrain Biopython version to <=1.80 so that `augur translate` is not broken by a deprecation of in newer versions. [#1152][] (@corneliusroemer)
```

That mentions the Augur command and a hint at the error message for users who run into the issue on a prior release. It doesn't mention BCBio.GFF because it's not relevant to the user here.